config/externalname.go

```diff
@@ -1155,6 +1155,18 @@ var ExternalNameConfigs = map[string]config.ExternalName{
 	"aws_redshift_hsm_configuration": config.ParameterAsIdentifier("hsm_configuration_identifier"),
 	// Redshift usage limits can be imported using the id
 	"aws_redshift_usage_limit": config.IdentifierFromProvider,
+	// Redshift Serverless Namespaces can be imported using the namespace_name
+	"aws_redshiftserverless_namespace": config.ParameterAsIdentifier("namespace_name"),
+	// Redshift Serverless Endpoint Access can be imported using the endpoint_name
+	"aws_redshiftserverless_endpoint_access": config.IdentifierFromProvider,
```

Comment:
Would `ParameterAsIdentifier` work here? According to the [docs](https://registry.terraform.io/providers/hashicorp/aws/4.67.0/docs/resources/redshiftserverless_endpoint_access#import), it should be possible to use the `endpoint_name` argument.